account_chatter/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Account Chatter
+===============
+
+This module allows to use chatter in CoA, Journals and Account Moves, to
+create audit changes in every model.
+
+To configure go to:
+
+1. Settings > Users > Account & Finance
+
+2. Check > Show account notifications
```

Comment:
Reading this, it looks (at least to me) that this is a global setting. To be more descriptive, I'd add an intermediate step mentioning that an user needs to be selected.

Or maybe clarifying that this a user-configurable feature.